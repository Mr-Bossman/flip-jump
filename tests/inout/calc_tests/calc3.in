23456-765
q
