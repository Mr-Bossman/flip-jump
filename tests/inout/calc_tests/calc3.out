>  23456-765
22691
>  q
